Code/Framework/AzToolsFramework/Platform/Windows/AzToolsFramework/API/EditorAssetSystemAPI_Windows.cpp

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+namespace AzToolsFramework
+{
+    namespace AssetSystem
+    {
+        const char* GetHostAssetPlatform()
+        {
+            return "pc";
```

Comment:
in the future might be nice to unify our platform name strings, maybe alongside the trait: `AZ_TRAIT_OS_IS_HOST_OS_PLATFORM`? And have PlatformDefaults also use them? or something like this?

```suggestion
            return AZ::PlatformDefaults::PlatformPC;
```

Though if someone were to change this to say upper case and that caused Asset lookups in the database to fail that'd be bad.

Reply:
well, in this case, the AP uses these to both auto enable platforms (in which case a enum would work) but also, to give the name of the actual folder in the cache (Cache/pc for example) to use.  Changing it to an enum would not work, we'd need another function to translate the enum to a folder name anyway (for backward compat).

But yeah, we should clean this all up in the future (no related to a build time improvement)